### ISSUE TITLE
It is crucial whether you open "SAFI_clean" in an excel or in Calc

### ISSUE THREAD

**Issue by lrskjr** [datacarpentry/spreadsheets-socialsci#114]
Hi!

I'm confused. When I open the file with Excel I get one result, when I open the file with CalC I get another result!

It may be a good idea to point out that this problem may occur.

I imagine that people like to use excel, because they ofte have that program installed, but they get a totally wrong result. 

As far as I can see, this is not a problem that a teacher can fix in a teaching situation, so it would be nice to be prepared on the situation.

I do not know why the problem occurs, but the result is that values ​​are missing in several columns, if you open the file with Excel. 

Im looking through the teaching material, because I'll be using it in Norway next month.

All the best
Lars  

<img width="264" alt="Udklip excel1" src="https://user-images.githubusercontent.com/24442392/89530594-10f70080-d7ef-11ea-93b3-cfdfe6069d4d.PNG">

<img width="360" alt="Udklip Calc1" src="https://user-images.githubusercontent.com/24442392/89530543-f886e600-d7ee-11ea-8e0d-a863a10197af.PNG">

**Comment by jd-foster** [comment]
I just want to note that my version of Excel has some settings for controlling automatic data conversion (on Mac) under Edit:
<img width="175" alt="image" src="https://github.com/user-attachments/assets/43ff97a3-6aae-48c9-847f-2553a1503da2">
<img width="620" alt="image" src="https://github.com/user-attachments/assets/02491ab1-e11b-4729-bb8f-76d67bb919a9">

**Comment by ndporter** [comment]
There was some discussion in maintainer meeting 20 Nov 2024 about whether there is a broader need to address both regional and software differences in the spreadsheet lessons and, if so, how. I'll try to link the notes once they're posted, though we didn't reach any conclusions other than "it's important to know".

**Comment by josenino95** [comment]
@allie-tatarian and I have been discussing the changes to fix the problem related to this issue (different results when opening the CSV with different software or regional configurations) and we'd appreciate you input @tobyhodges @ErinBecker. We think we should do one of the following:

1. In [Ep 5. Quality assurance](https://datacarpentry.github.io/spreadsheets-socialsci/04-quality-assurance.html) include a section at the beginning explaining how to import "correctly" a CSV to Excel/Calc
2. Replace in Ep 5 the use of the clean SAFI csv for an xlsx that contains the same data. And later, include in Ep 6 (where csvs are actually discussed) a section on how to import a csv

My preference is option 2, as that makes mores sense with the flow of the lesson, talking about csvs at the end. However, this has the downside of not using the [official SAFI DC-social science data](https://datacarpentry.github.io/socialsci-workshop/#data), which doesn't have an xlsx version of the clean data. We'd need to create the xlsx version ourselves and upload it to the lesson GitHub repo.

**Comment by ndporter** [comment]
Thanks for working on this sticky issue @josenino95 and @allie-tatarian ! FWIW my slight preference as an instructor is for option 1, because it resembles typical workflows more closely (get data in whatever format someone sends and figure out how to import it correctly). But either does work.

**Comment by tobyhodges** [comment]
I agree with @ndporter that option 1 is preferable. Perhaps, to address your concern about CSV not being addressed until episode 6, we could move the brief description of the format from that episode into ep 5? E.g. this sentence from that ep 6:

> CSV files are plain text files where the columns are separated by commas, hence ‘comma separated values’ or CSV.

And supplement it with one more, to explain why we might be importing from this format, e.g.

> CSV is a format commonly used for tabular data, which we will discuss further in the next episode.

These are relatively small changes to the flow of the lesson and I think they can be made relatively safely.

**Comment by josenino95** [comment]
Thank you all for your input. With the merge of PR #186 as now it's more clear in the lesson how to open a csv in Excel and Calc to have consistent results (with screenshots)
